CMakeLists.txt

```diff
@@ -116,8 +116,13 @@ add_executable(sexpr-wasm ${SEXPR_WASM_SRCS})
 add_dependencies(everything sexpr-wasm)
 
 # hexfloat-test
+option(BUILD_TESTS "Build GTest-based tests" ON)
```

Comment:
oh wait, my bad, the outer if replaces the previous if. So yeah i think this works.